lib/index.js

```diff
@@ -0,0 +1,110 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const assert = require('assert-plus');
+const bedrock = require('bedrock');
+const {util: {clone, BedrockError}} = bedrock;
+
+const packages = [];
```

Comment:
Is it worth adding any magic to more strictly ensure there is only one of these package mangers loaded globally?  Could be done later.  Just seems to come up in other packages like bedrock where, mostly during development with symlinks, you register for events on the wrong bedrock instance.  I imagine that could happen with this package too.

Reply:
That is frustrating.  Haven't been burned by that issue in a long while.  I don't know what magic you refer to, but I agree that it can be post 1.0 release.